index.html

```diff
@@ -106,7 +106,7 @@ <h1 class="brackets-logo">
         <div class="large-12 columns">
             <h1><span data-i18n="index.page.hero.description">A modern, open source text editor that understands web design.</span></h1>
             <div id="download">
-                <a id="hero-cta-button" href="https://github.com/adobe/brackets/releases/latest" class="large rounded radius button"><div data-i18n="[html]index.page.hero.download">Download Brackets <d id="download-brackets-version">1.1</d></div>
+                <a id="hero-cta-button" href="https://github.com/adobe/brackets/releases/latest" class="large rounded radius button"><div data-i18n="[html]index.page.hero.download">Download Brackets <d id="download-brackets-version">1.2</d></div>
```

Comment:
It seems confusing if the button says "1.2" but clicking it on Linux still downloads 1.1 -- even if this is just a temporary arrangement for a few days.

It seems simple to rewrite the button label string onload, but even that seems confusing.  We have no way of withholding update notifications for Linux, so everyone will come to the website right away...  Could we do something similar to `updateAlertContent()` to show a banner to Linux users?  That way we can explain why the 1.2 notification disagrees with what's available on the website, and give an ETA for when that will change (especially important since there will be _no_ later notification for them when the Linux build becomes available).